airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/orchestrator/DbtJobOrchestrator.java

```diff
@@ -77,11 +77,10 @@ public Optional<String> runJob() throws Exception {
         Math.toIntExact(jobRunConfig.getAttemptId()),
         workerConfigs.getResourceRequirements(),
         new DbtTransformationRunner(
-            processFactory, NormalizationRunnerFactory.create(
-                destinationLauncherConfig.getDockerImage(),
+            processFactory, new DefaultNormalizationRunner(
                 processFactory,
-                NormalizationRunnerFactory.NORMALIZATION_VERSION,
-                destinationLauncherConfig.getNormalizationDockerImage())));
+                destinationLauncherConfig.getNormalizationDockerImage(),
+                destinationLauncherConfig.getIntegrationType())));
```

Comment:
This is a field that indicates which type of dialect should be taken during normalization. Due to this, the values of the length of column and table names, casting types, etc. will be determined. Previously, this type was defined using `Map`, which was located in `NormalizationRunnerFactory`. But since we decided to abandon the `NormalizationRunnerFactory` and separate the normalization and the platform, it is better to keep this value in the definition file, and pass it to the actor_definition table

Reply:
I also renamed `integrationType` to `normalizationIntegrationType`